### PR TITLE
ADD: show fingerprint and derivation path for HD wallets

### LINKED ISSUE #2823: Add support for fp & path in all HD wallets
should also be rendered in wallet -> details screen

### PR DESCRIPTION
**Pull request by limpbrains**
- fingerprint shown for HDLegacyP2PKHWallet, HDSegwitBech32Wallet, HDSegwitP2SHWallet and for WatchOnly+zpub wallets
- derivation path shown for all HD wallets

closes #2823

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8rm0upg7my0u16bx2rzqungqmg

[download apk](http://filestorage.bluewallet.io:1488/limpbrains-show-fp-1616516578.apk)

**Review by Overtorment — APPROVED** [review]

File: class/wallets/multisig-hd-wallet.js [hunk 1]
```diff
@@ -189,7 +185,7 @@ export class MultisigHDWallet extends AbstractHDElectrumWallet {
     } else {
       // mnemonics. lets derive fingerprint (if it wasnt provided)
       if (!bip39.validateMnemonic(key)) throw new Error('Not a valid mnemonic phrase');
-      fingerprint = fingerprint || MultisigHDWallet.seedToFingerprint(key);
+      fingerprint = fingerprint || AbstractHDElectrumWallet.seedToFingerprint(key);
```
[the remark this answers is not in the document]
Comment (limpbrains) [reply class/wallets/multisig-hd-wallet.js]: fixed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ztthwt3gm1320rx172hnxbmcyw

[download apk](http://filestorage.bluewallet.io:1488/limpbrains-show-fp-1616598895.apk)

**Review by Overtorment — APPROVED** [review]

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**